src/main/java/io/cdap/plugin/snowflake/sink/batch/StructuredRecordToCSVRecordTransformer.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.snowflake.sink.batch;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import io.cdap.cdap.api.data.format.StructuredRecord;
+import io.cdap.cdap.api.data.schema.Schema;
+import io.cdap.cdap.format.StructuredRecordStringConverter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Transforms a {@link StructuredRecord} to a {@link CSVRecord}
+ */
+public class StructuredRecordToCSVRecordTransformer {
+  private static final Logger LOG = LoggerFactory.getLogger(StructuredRecordToCSVRecordTransformer.class);
+
+  public CSVRecord transform(StructuredRecord record) throws IOException {
+    List<String> fieldNames = new ArrayList<>();
+    List<String> values = new ArrayList<>();
+
+    for (Schema.Field field : record.getSchema().getFields()) {
+      String fieldName = field.getName();
+      String value = convertSchemaFieldToString(record.get(fieldName), field, record);
+
+      fieldNames.add(fieldName);
+      values.add(value);
```

Comment:
what would happen when null value is added?

Reply:
Just tested with by putting 
```String value = null;```
instead.

The pipeline run was successful in both source and sink. In result tables all values set to NULL as expected.

Reply:
this case should be added to integration test

Reply:
Done.